### ISSUE TITLE
[BUG]After Update Chia Blockchain 1.2.11 will not connect

### ISSUE THREAD

**Issue by kijanajambazi** [Chia-Network/chia-blockchain#9168]
After updating to 1.2.11 my chia wallet will not connect even after 20 hours and counting. I've included my debug log below, thanks.

- OS: Windows 
- OS 10
- CPU: Intel Xeon i910900

![image](https://user-images.githubusercontent.com/93840427/140622028-ff8cd082-6f58-470b-b310-3069f9908c19.png)

[debug.log](https://github.com/Chia-Network/chia-blockchain/files/7491127/debug.log)

**Comment by kijanajambazi** [comment]
Rebooted, launched GIU and waited 3-5 minutes....please see new DB log, thanks.

[debug.log](https://github.com/Chia-Network/chia-blockchain/files/7491173/debug.log)
..

**Comment by Rekternino** [comment]
Same problem I have deleted db file it was 30tb. Right now it's syncing. But it is a really big problem. I will wait for 1 day for sync. My farming stopped.

**Comment by bob4tmp** [comment]
Same problem (

**Comment by loppefaaret** [comment]
> Rebooted, launched GIU and waited 3-5 minutes....please see new DB log, thanks.
> 
> [debug.log](https://github.com/Chia-Network/chia-blockchain/files/7491173/debug.log) ..

this log shows the `full_node` starting the initialize the blockchain about 5 minutes after launch
are you still seeing spinning circles in the "Full Node" tab in the GUI ?

**Comment by kijanajambazi** [comment]
@loppefaaret , you're correct it did finally launch after about 10minutes. Thank you very much for the recommendation and follow-up, all's good!
